libraries/AP_Camera/AP_Camera.h

```diff
@@ -115,6 +115,9 @@ class AP_Camera {
     void take_picture();
     void take_picture(uint8_t instance);
 
+    // take multiple pictures
+    void take_multiple_pictures(const float& time, const float& total_pictures, const float& seq);
```

Comment:
let's name the argument "interval" or "time_interval".

Also rename, "total_pictures" to "total_num".  I don't think we should support the "seq" number in the case of multiple images.

>> label="Sequence Number" minValue="1" increment="1">Capture sequence number starting from 1. This is only valid for single-capture (param3 == 1), otherwise set to 0. Increment the capture ID for each capture command to prevent double captures when a command is re-transmitted.</param>